### ISSUE TITLE
Cryptonight-FireIce\xmr-stak.exe cryptonightv7 not cryptonight_v7

### ISSUE THREAD

**Issue by jestercore** [MultiPoolMiner/MultiPoolMiner#1765]
There is wrong commandline for xmr-stak for AMD, cryptonightv7 not cryptonight_v7 like miner asks
![issue](https://user-images.githubusercontent.com/33821148/40578360-06e477ba-613d-11e8-983a-6f61a70e6fcd.png)

**Comment by paulpoco** [comment]
Yes there is an error.

I downloaded latest Master and as it starts Benchmarking it lists CryptoNightV7 but it doesn't finish the Benchmark, and after all the CPU miners were done, CryptoNightV7 wasn't listed.

**Comment by jestercore** [comment]
So, FireiceAmd.ps1, FireiceCpu.ps1, and FireiceNvidia.ps1 should have this lines

```
$Commands = [PSCustomObject]@{
    # "cryptonight" = "" #CryptoNight is ASIC territory
    "cryptonight_lite" = "" # **CryptoNight_Lite**
    "cryptonightV7" = "" #**CryptoNight_V7**
```

not like this

```
$Commands = [PSCustomObject]@{
    # "cryptonight" = "" #CryptoNight is ASIC territory
    "cryptonight_lite" = "" # CryptoNight-Lite
    "cryptonightV7" = "" #CryptoNightV7
```

**Comment by UselessGuru** [comment]
@jestercore  Not quite ;-)

Fixed in #1766
